src/auth/negotiate/kerberos/negotiate_kerberos_pac.cc

```diff
@@ -362,6 +367,114 @@ getextrasids(char *ad_groups, uint32_t ExtraSids, uint32_t SidCount)
     return ad_groups;
 }
 
+static char *
+get_resource_group_domain_sid(uint32_t ResourceGroupDomainSid) {
+
+    if (ResourceGroupDomainSid!= 0) {
+        uint8_t rev;
+        uint64_t idauth;
+        char dli[256];
+        char *ag;
+        int l;
+
+        align(4);
+
+        uint32_t nauth = get4byt();
+
+        size_t length = 1+1+6+nauth*4;
```

Comment:
Please document what characters the magic numbers are reserving space for.

Reply:
I commented this string in a new file version:
// length = revision[1byte]+nsub[1byte]+idauth[6bytes]+nauth*sauth[4bytes]
size_t length = 1+1+6+nauth*4;